src/library/librarycontrol.cpp

```diff
@@ -60,114 +62,89 @@ LibraryControl::LibraryControl(Library* pLibrary)
     m_numSamplers.connectValueChanged(SLOT(slotNumSamplersChanged(double)));
     m_numPreviewDecks.connectValueChanged(SLOT(slotNumPreviewDecksChanged(double)));
 
-    // Controls to navigate vertically within currently focussed widget (up/down buttons)
```

Comment:
I look again and check what happened but your changes didn't give me any conflicts